srml/contract/src/lib.rs

```diff
@@ -390,6 +390,10 @@ decl_storage! {
 		pub CodeStorage: map CodeHash<T> => Option<wasm::PrefabWasmModule>;
 		/// The storage items associated with an account/key.
 		StorageOf: double_map T::AccountId, blake2_256(Vec<u8>) => Option<Vec<u8>>;
```

Comment:
I did expect this pr to be merged before the contract switch to child trie: this double_map should not be use anymore.

Reply:
oups yes